### PR TITLE
cmd: add more logs to debug cdc data-dir set up.

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve? 

This PR add log to help debug data-dir related issue.

### Check List 

Tests 

 - No code

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Release note

No release note

### CODE REVIEW

File: cdc/server.go [hunk 1]
```diff
@@ -263,8 +263,8 @@ func (s *Server) initDataDir(ctx context.Context) error {
 		return errors.Trace(err)
 	}
 
-	log.Info(fmt.Sprintf("%s is set as data-dir (%dGB available), ticdc recommend disk for data-dir "+
-		"at least have %dGB available space", conf.DataDir, diskInfo.Avail, dataDirThreshold))
+	log.Info(fmt.Sprintf("%s is set as data-dir (%dGB available), sort-dir=%s. "+
+		"ticdc recommend disk for data-dir at least have %dGB available space", conf.DataDir, diskInfo.Avail, conf.Sorter.SortDir, dataDirThreshold))
```
Comment (liuzix) [inline cdc/server.go:267]:
```suggestion
		"It is recommended that the disk for data-dir at least have %dGB available space", conf.DataDir, diskInfo.Avail, conf.Sorter.SortDir, dataDirThreshold))
```

File: cmd/server.go [hunk 2]
```diff
@@ -147,6 +147,14 @@ func loadAndVerifyServerConfig(cmd *cobra.Command) (*config.ServerConfig, error)
 		if err := strictDecodeFile(serverConfigFilePath, "TiCDC server", conf); err != nil {
 			return nil, err
 		}
+		// user specified sort-dir should not take effect, it's always `/tmp/sorter`
+		// if user try to set sort-dir by config file, warn it.
+		if conf.Sorter.SortDir != config.DefaultSortDir {
+			cmd.Printf(color.HiYellowString("[WARN] --sort-dir is deprecated in server settings. " +
+				"sort-dir will be set to `{data-dir}/tmp/sorter`. The sort-dir here will be no-op\n"))
+
+			conf.Sorter.SortDir = config.DefaultSortDir
```
Comment (overvenus) [inline cmd/server.go:156]: @leoppro PTAL

**Review by overvenus — COMMENTED** [review]
Code LGTM

File: cmd/server.go (hunk 2 above)
Comment (leoppro) [reply cmd/server.go:156]: LGTM

**Review by overvenus — APPROVED** [review]

**Review by amyangfei — COMMENTED** [review]
This PR sets `conf.Sorter.SortDir` in three places, one question, will `conf.Sorter.SortDir` be always reset to `filepath.Join(conf.DataDir, config.DefaultSortDir)` after cdc server calls `initDataDir`

**Comment by 3AceShowHand** [comment]
> This PR sets `conf.Sorter.SortDir` in three places, one question, will `conf.Sorter.SortDir` be always reset to `filepath.Join(conf.DataDir, config.DefaultSortDir)` after cdc server calls `initDataDir`

yes, it's guaranteed that `conf.Sorter.SortDir` will be set to `filepath.Join(conf.DataDir, config.DefaultSortDir)` after cdc server calls `initDataDir`

only conf.DataDir will be decided by user-specified value or existed changefeed information. `conf.Sort.SortDir` will never change by user.

**Review by amyangfei — APPROVED** [review]

**Pull request merged**